src/Bindings/DeprecatedBindings.cpp

```diff
@@ -599,6 +599,13 @@ void DeprecatedBindings::Bind(lua_State * tolua_S)
 		tolua_function(tolua_S, "UpdateSign", tolua_cWorld_SetSignLines);
 	tolua_endmodule(tolua_S);
 
+
+	tolua_beginmodule(tolua_S, "cNoteEntity");
+		tolua_function(tolua_S, "GetPitch", tolua_AllToLua_cNoteEntity_GetNote00);
+		tolua_function(tolua_S, "SetPitch", tolua_AllToLua_cNoteEntity_SetNote00);
+		tolua_function(tolua_S, "IncrementNote", tolua_AllToLua_cNoteEntity_IncrementNote00);
```

Comment:
You can't "redirect" to a handler in automatic bindings - they are marked `static` so they are unavailable from other source files; also we want to output a warning and stacktrace to the console if these old functions are used.

Instead, create new functions for these that remaps the call:
```cpp
static int tolua_cNoteEntity_GetPitch(lua_State * tolua_S)
{
	cLuaState LuaState(tolua_S);

	if (
		!LuaState.CheckParamUserType(1, "cNoteEntity") ||
		!LuaState.CheckParamEnd(2)
	)
	{
		return 0;
	}

	cNoteEntity * Self = nullptr;

	if (!LuaState.GetStackValues(1, Self))
	{
		tolua_error(LuaState, "Failed to read parameters", nullptr);
	}
	if (Self == nullptr)
	{
		tolua_error(LuaState, "invalid 'self' in function 'GetNote'", nullptr);
	}
	LuaState.Push(Self->GetNote());
	LOGWARNING("Warning: 'cNoteEntity:GetPitch' function is deprecated. Please use 'cNoteEntity:GetNote' instead.");
	LuaState.LogStackTrace(0);
	return 1;
}
```

Also, please alpha-sort both the functions in a module, and module by name.